rest-api-spec/src/main/resources/rest-api-spec/test/scripts/20_get_script_context.yml

```diff
@@ -4,24 +4,20 @@
       reason: "get_all_contexts introduced in 7.6.0"
   - do:
       get_script_context: {}
-  - match: { contexts.aggregation_selector: {} }
-  - match: { contexts.aggs: {} }
-  - match: { contexts.aggs_combine: {} }
-  - match: { contexts.aggs_init: {} }
-  - match: { contexts.aggs_map: {} }
-  - match: { contexts.aggs_reduce: {} }
-  - match: { contexts.bucket_aggregation: {} }
-  - match: { contexts.field: {} }
-  - match: { contexts.filter: {} }
-  - match: { contexts.ingest: {} }
-  - match: { contexts.interval: {} }
-  - match: { contexts.number_sort: {} }
-  - match: { contexts.processor_conditional: {} }
-  - match: { contexts.score: {} }
-  - match: { contexts.script_heuristic: {} }
-  - match: { contexts.similarity: {} }
-  - match: { contexts.similarity_weight: {} }
-  - match: { contexts.string_sort: {} }
-  - match: { contexts.template: {} }
-  - match: { contexts.terms_set: {} }
-  - match: { contexts.update: {} }
+  - match: { contexts.0.name: "aggregation_selector" }
+  - match: { contexts.0.methods: [{"name": "execute", "return_type": "boolean", "params": []}, {"name": "getParams", "return_type": "java.util.Map", "params": []}] }
+
+  - match: { contexts.1.name: "aggs" }
```

Comment:
Why is this structure changed to a list of objects? The order of these do not matter, so I don't think it should be a list.

Reply:
Agreed, I was going with ease of using our existing parsing primitives, which have a difficulty addressing dynamic keys.  Will take a pass at making it a map and we'll see how ugly it gets.

Reply:
It got pretty ugly.  Per offline conversation, we're gonna keep this structure for now.

Reply:
It got pretty ugly doing the fragment parsing and made testing difficult. Per offline conversation, we're gonna keep this structure for now.